docs/glossary.asciidoc

```diff
@@ -423,3 +485,10 @@ A workspace where you build presentations of your live data in <<glossary-canvas
 See
 {kibana-ref}/canvas.html[Create a workpad].
 // end::workpad-def[]
+
+[[glossary-wms]] WMS ::
```

Comment:
Rather than defining this acronym in the description, maybe it's better to define it in the first usage:
```suggestion
[[glossary-wms]] Web Map Service (WMS) ::
```